Jellyfin.Data/Entities/MediaSegment.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+///     An entity representing the metadata for a group of trickplay tiles.
+/// </summary>
+public class MediaSegment
+{
+    /// <summary>
+    ///     Gets or sets the id of the media segment.
+    /// </summary>
+    [JsonIgnore]
+    public Guid Id { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the id of the associated item.
+    /// </summary>
+    [JsonIgnore]
+    public Guid ItemId { get; set; }
```

Comment:
Those are no longer needed now that you have a separate DTO right?

```suggestion
    public Guid Id { get; set; }

    /// <summary>
    ///     Gets or sets the id of the associated item.
    /// </summary>
    public Guid ItemId { get; set; }
```